src/Views/ShellPage.xaml

```diff
@@ -20,6 +20,7 @@
             IsBackButtonVisible="Collapsed"
             SelectedItem="{x:Bind ViewModel.Selected, Mode=OneWay}"
             IsSettingsVisible="True"
+            OpenPaneLength="350"

```

Comment:
Can we instead define the content of the `NavigationViewItem` in `ShellPage.xaml.cs`?
```xml
<NavigationViewItem Icon="Accept">
    <TextBlock TextWrapping="NoWrap" TextTrimming="CharacterEllipsis">Hello world</TextBlock>
</NavigationViewItem>
```

Reply:
We could, but not right now.  What would the benefit of this be?

Reply:
If I understood the issue correctly, we don't want to clip the text in the navigation view if the text cannot fit in the available space. What I am proposing above is, would it be possible instead of resizing the width of the navigation panel, can we show ellipsis `...` (maybe with the option of a tooltip when trimmed)?

Reply:
I understand your point.  I wonder if ellipses still goes against craftmanship reviews because not all the text exists?

https://github.com/microsoft/devhome/issues/1820 is an issue to track the effort to copy over some of the NavigationViewItem code to get text trimming.

Reply:
I've heard it said (but not from an accessibility expert, so grain of salt) that making cut off text visible in a tooltip does NOT meet accessibility standards.